sbt-dotty/sbt-test/sbt-dotty/example-project/test

```diff
@@ -1,4 +1,5 @@
 > run
+> doc
 > 'set initialCommands := "1 + 1" '
 # FIXME: does not work on the CI
```

Comment:
What's preventing this from being run on the CI? It would be nice to also check that this generates what it's supposed to.

Reply:
The REPL doesn't work in the CI. JLine doesn't work outside of a "real terminal"

Reply:
By default JLine spawns a "dumb" terminal if not able to create a terminal. But you can't do anything with it, not sure we can even use it for test purpose. So i disabled it:
https://github.com/lampepfl/dotty/blob/0d7826c8a77bfa0da727ac21213475739faa9428/compiler/src/dotty/tools/repl/JLineTerminal.scala#L21